src/lib/y2network/autoinst_profile/networking_section.rb

```diff
@@ -34,26 +35,41 @@ module AutoinstProfile
     #  </networking>
     #
     # @see RoutingSection
-    class NetworkingSection
-      # @return [Boolean]
-      attr_accessor :setup_before_proposal
-      # @return [Boolean]
-      attr_accessor :start_immediately
-      # @return [Boolean]
-      attr_accessor :keep_install_network
-      # @return [Integer]
-      attr_accessor :strict_ip_check_timeout
+    class NetworkingSection < SectionWithAttributes
+      def self.attributes
+        [
+          { name: :setup_before_proposal },
+          { name: :start_immediately },
+          { name: :keep_install_network },
+          { name: :strict_ip_check_timeout },
+          { name: :routing },
+          { name: :dns },
+          { name: :interfaces },
+          { name: :udev_rules },
+          { name: :s390_devices }
+        ]
+      end
+
+      define_attr_accessors
 
-      # @return [RoutingSection]
-      attr_accessor :routing
-      # @return [DNSSection]
-      attr_accessor :dns
-      # @return [InterfacesSection]
-      attr_accessor :interfaces
-      # @return [UdevRulesSection]
-      attr_accessor :udev_rules
-      # @return [S390DevicesSection]
-      attr_accessor :s390_devices
+      # @!attribute setup_before_proposal
+      #  @return [Boolean]
+      # @!attribute start_immediately
+      #  @return [Boolean]
+      # @!attribute keep_install_network
+      #  @return [Boolean]
+      # @!attribute strict_ip_check_timeout
+      #  @return [Boolean]
+      # @!attribute routing
+      #   @return [RoutingSection]
+      # @!attribute dns
+      #   @return [DNSSection]
+      # @!attribute interfaces
```

Comment:
We probably should modify it as it is really a list of `InterfaceSection` and the same could be applied to other attributes but for the time being do not plan to change it.